packages/editor/src/client/blocks/linkPageBreakBlock.tsx

```diff
@@ -13,46 +34,218 @@ export function LinkPageBreakBlock({
   autofocus,
   disabled
 }: LinkPageBreakBlockProps) {
-  const {text, linkText, linkURL} = value
+  const {
+    text,
+    linkText,
+    linkURL,
+    styleOption,
+    layoutOption,
+    richText,
+    linkTarget,
+    hideButton,
+    image,
+    templateOption
+  } = value
   const focusRef = useRef<HTMLTextAreaElement>(null)
+  const focusInputRef = useRef<HTMLInputElement>(null)
+
+  /* eslint-disable i18next/no-literal-string */
+  /* eslint-disable @typescript-eslint/no-non-null-assertion */
 
-  const {t} = useTranslation()
+  // const {t} = useTranslation()
```

Comment:
Please add all language translations to the `en.json`

Reply:
could you or your team do it?

Reply:
Nope.